src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -12,9 +12,12 @@ struct cobj_header *hs[MAX_NUM_SPDS + 1];
 /* The booter uses this to keep track of each comp */
 struct comp_cap_info {
 	struct cos_compinfo *compinfo;
+	/* TODO: figure out a better way to do this. Need one ST_user_cap per linked symb. Currently just hard coding 5. */
+	struct usr_inv_cap   ST_user_caps[5];
```

Comment:
I think `usr_invinfo` for the name of the struct would be better. `usr_inv_cap` doesn't seem to cover the scope of what the struct holds. :) 
+ `ST_` - capital letters in the variable name is very uncommon in C programming.